Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:18.04
```

Comment:
I used Defects4J in my experiment which runs on Ubuntu 18. I have changed the Dockerfile for using Ubuntu 20.04, as it might be better to use the latest LTS version.